xarray/util/generate_aggregations.py

```diff
@@ -312,12 +321,21 @@ def generate_method(self, method):
 
         yield TEMPLATE_RETURNS.format(**template_kwargs)
 
-        yield TEMPLATE_SEE_ALSO.format(
-            **template_kwargs,
-            docref=self.docref,
-            docref_description=self.docref_description,
-            see_also_obj=self.see_also_obj,
-        )
+        if method.name == "count":
```

Comment:
```suggestion
       template = _COUNT_SEE_ALSO if method.name == "count" else TEMPLATE_SEE_ALSO
```
And then `template.format(...`

Reply:
Thanks for the suggestion! Just committed it along with the whats-new entry